src/plugins/intel_cpu/src/plugin.h

```diff
@@ -63,6 +63,19 @@ class Engine : public InferenceEngine::IInferencePlugin {
     const std::string deviceFullName;
 
     std::shared_ptr<void> specialSetup;
+
+#if defined(OV_CPU_WITH_ACL)
+    struct SchedulerGuard {
+        SchedulerGuard();
+        ~SchedulerGuard();
+        static std::shared_ptr<SchedulerGuard> instance();
+        static std::mutex mutex;
```

Comment:
Isn't single mutex is enough?

Reply:
@dmitry-gorokhov removed additional mutex